spec/miscellaneous/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -97,3 +97,81 @@
     end
   end
 end
+
+describe "rake After:add_user_id_to_kudos" do
+  context "when user has no pseud" do
+    it "runs without erroring or updating user_id or pseud_id" do
+      pseudless_user = create(:user)
+      deleted_pseud_kudos = create(:kudo,
+                                   pseud_id: pseudless_user.default_pseud_id)
+
+      pseudless_user.default_pseud.delete
+      pseudless_user.reload
+      deleted_pseud_kudos.reload
+
+      subject.invoke
+
+      deleted_pseud_kudos.reload
+      expect(deleted_pseud_kudos.user_id).to be(nil)
+      expect(deleted_pseud_kudos.pseud_id).not_to be(nil)      
```

Comment:
We should save the pseud ID before deleting the pseud, then compare to the saved ID here.